crates/build/src/lib.rs

```diff
@@ -5,13 +5,29 @@
 use anyhow::{bail, Context, Result};
 use path_absolutize::Absolutize;
 use spin_loader::local::config::{RawAppManifest, RawComponentManifest};
+use std::fs::read_to_string;
 use std::path::{Path, PathBuf};
 use subprocess::{Exec, Redirection};
+use toml::Value;
 use tracing::log;
 
 /// If present, run the build command of each component.
 pub async fn build(app: RawAppManifest, src: &Path) -> Result<()> {
     let src = src.absolutize()?;
+    let mut path = src.to_str().unwrap().split('/').collect::<Vec<&str>>();
+    path.pop();
+    path.push("spin.toml");
+    let path = path.join("/");
+
+    let spin_manifest = read_to_string(&path).unwrap();
+    let manifest_info: Value = toml::from_str(&spin_manifest).unwrap();
```

Comment:
This looks like it re-parses the TOML file.  You shouldn't need to do this because the `RawAppManifest` passed into this function contains this pre-parsed.

Reply:
I should have checked if you were already parsing the `.toml` files. sorry.